tests/Satooshi/Bundle/CoverallsV1Bundle/Repository/JobsRepositoryTest.php

```diff
@@ -35,10 +35,14 @@ protected function createApiMockWithRequirementsNotSatisfiedException()
             'dumpJsonFile',
             'send',
         );
-        $api = $this->getMockBuilder('Satooshi\Bundle\CoverallsV1Bundle\Api\Jobs')
-        ->disableOriginalConstructor()
-        ->setMethods($jobsMethods)
-        ->getMock();
+        if (method_exists(__CLASS__, 'createPartialMock')) {
```

Comment:
I don't like the condition to make a mock.
If we would use `->prophesize`, would it work for this case ?